src/process/Convert.js

```diff
@@ -14,6 +14,7 @@ class Convert {
 
         let validation = this.schemaValidator.validate(output);
         if (!validation.valid) {
+            console.error(validation);
```

Comment:
Instead of logging here, it feels like it should be done in `main.js`. You could create a new Error type, and pass the validation error along e.g.

`throw new ValidationError("Converted author schema is not valid EQ schema", validation)`

You already wrap in a try/catch in `main.js`, so seems like the ideal place to handle logging